src/pages/PoolDetails/PoolDetailsStats.tsx

```diff
@@ -0,0 +1,229 @@
+  const currencies = [
+    useCurrency(poolData?.token0?.id, chainId) ?? undefined,
+    useCurrency(poolData?.token1?.id, chainId) ?? undefined,
+  ]
+
+  // We can't wrap this in a useMemo hook because useColor is also a hook
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const color0 = useColor(currencies[0]?.wrapped)
+  const color1 = useColor(currencies[1]?.wrapped)
+
+  const [token0, token1] = useMemo(() => {
+    const fullWidth = poolData?.tvlToken0 / poolData?.token0Price + poolData?.tvlToken1
+    const token0FullData = {
+      ...poolData?.token0,
+      price: poolData?.token0Price,
+      tvl: poolData?.tvlToken0,
+      color: color0,
+      percent: poolData?.tvlToken0 / poolData?.token0Price / fullWidth,
+      currency: currencies[0],
+    }
+    const token1FullData = {
+      ...poolData?.token1,
+      price: poolData?.token1Price,
+      tvl: poolData?.tvlToken1,
+      color: color1,
+      percent: poolData?.tvlToken1 / fullWidth,
+      currency: currencies[1],
+    }
+    return isReversed ? [token1FullData, token0FullData] : [token0FullData, token1FullData]
+  }, [color0, color1, currencies, isReversed, poolData])
+
+  return (
+    <StatsWrapper>
+      <HeaderText>
+        <Trans>Stats</Trans>
+      </HeaderText>
+      <StatItemColumn>
+        <ThemedText.BodySecondary>
+          <Trans>Pool balances</Trans>
+        </ThemedText.BodySecondary>
+        <PoolBalanceSymbols>
+          <PoolBalanceTokenNames>
+            {!screenIsNotLarge && (
+              <CurrencyLogo currency={token0.currency} size="20px" style={{ marginRight: '8px' }} />
+            )}
+            {formatNumber({
+              input: token0.tvl,
+              type: NumberType.TokenNonTx,
+            })}
+            &nbsp;
+            {token0.symbol}
+          </PoolBalanceTokenNames>
+          <PoolBalanceTokenNames>
+            {!screenIsNotLarge && (
+              <CurrencyLogo currency={token1.currency} size="20px" style={{ marginRight: '8px' }} />
+            )}
+            {formatNumber({
+              input: token1.tvl,
+              type: NumberType.TokenNonTx,
+            })}
+            &nbsp;
+            {token1.symbol}
+          </PoolBalanceTokenNames>
+        </PoolBalanceSymbols>
+        {screenIsNotLarge && (
+          <Row data-testid="pool-balance-chart">
+            {token0.color && <BalanceChartSide percent={token0.percent} $color={token0.color} isLeft={true} />}
+            {token1.color && <BalanceChartSide percent={token1.percent} $color={token1.color} isLeft={false} />}
+          </Row>
+        )}
+      </StatItemColumn>
+      <StatItem title={<Trans>TVL</Trans>} value={poolData.tvlUSD} delta={poolData.tvlUSDChange} />
+      <StatItem title={<Trans>24H volume</Trans>} value={poolData.volumeUSD} delta={poolData.volumeUSDChange} />
+      <StatItem title={<Trans>24H fees</Trans>} value={poolData.volumeUSD * (poolData.feeTier / 1000000)} />
```

Comment:
double-checking, should we divide by `10000` here?

Reply:
Yes to get an accurate calculation. This is the same logic currently used on the info site.